terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/codelist/CodeListTest.java

```diff
@@ -594,4 +597,20 @@ public void test11_04_form() {
         driver.findElement(By.id("btnback")).click();
     }
 
+    @Test
+    public void test11_05_form() {
+        driver.findElement(By.linkText("German")).click();
+        driver.findElement(By.id("codelist_11_05")).click();
+
+        assumeThat(driver.findElement(By.id("defaultLocale")), is(
+                "The default locale is " + Locale.JAPAN));
+        assertThat(driver.findElement(By.cssSelector("option[value=\"key1\"]"))
+                .getText(), is("ラベル1"));
+        assertThat(driver.findElement(By.cssSelector("option[value=\"key2\"]"))
+                .getText(), is("ラベル2"));
+        assertThat(driver.findElement(By.cssSelector("option[value=\"key3\"]"))
+                .getText(), is("ラベル3"));
```

Comment:
- [ ] Assuming a bean definition, You can make succeed test when default locale is `ja`, `ja_JP`, `en`, `en_US` etc...
You should change to apply this. ex...

* Controller
Set default language locale.
```java
Locale defaultLocale = Locale.getDefault();
if (defaultLocale.getCountroy().length > 0) {
  defaultLocale = new Locale(defaultLocale.getLanguage());
}
model.addAttribute("defaultLocale", defaultLocale);
```
* JUnit
Assume when default locale is `jp` or `en`.
```java
String textDefaultLocale = driver.findElement(By.id("defaultLocale")).getText();
assumeThat(textDefaultLocale ,
  anyOf(containsString(Locale.JAPANESE), containsString(Locale.ENGLISH));
if (textDefaultLocale.contains(Locale.JAPANESE) {
  // assert Japanese labels.
} else {
  // assert English labels.
}
```

It seems also good to remove `en` codelist from `CL_CODELIST11_05` and make it simple.

Reply:
I excluded the implementation related to `test11_05_form` and gave `@Ignore` for the test.
a6c96bda709fdf5f46917c7ca0efd9ebec822f73

Note : There is a possibility that application startup may fail if implemented.
　　　Also, we implemented unit test of this function with terasoluna-gfw.
　　　Therefore, correspondence was made.